module1/homework/calculate/calculate.hpp

```diff
@@ -2,6 +2,23 @@
 #include <string>
 
 std::string calculate(const std::string& command, int first, int second) {
-    // TODO: Implement your solution here and return proper value
-    return "";
+    std::string result = "Invalid data";
+
+    if( command == "add" ) { 
+      result = std::to_string(first+second);
+    } else
+        if( command == "subtract" ) { 
```

Comment:
it's a good idea to keep "else if" keywords in the same line
```suggestion
        else if (command == "subtract") { 
```